qa/objectstore/bluestore.yaml

```diff
@@ -20,3 +20,21 @@ overrides:
         osd failsafe full ratio: .95
 # this doesn't work with failures bc the log writes are not atomic across the two backends
 #        bluestore bluefs env mirror: true
+  ceph-deploy:
+    fs: xfs
+    bluestore: yes
```

Comment:
Thats true and currently thats the case (implicitly ceph-disk is using bluestore option).  Adding that option to filestore-xfs and adding to ceph-deploy options will increase the tests from 24 to 48, Do you want to run those filestore tests as well for master?